packages/shared/src/settings/margin.ts

```diff
@@ -0,0 +1,73 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Bundle, SettingBlock } from '@frontify/guideline-blocks-settings';
+import { appendUnit } from '../helpers/settings/appendUnit';
+import { numericalOrPixelRule } from '../utilities/rules/numericalOrPixelRule';
+import { presetCustomValue } from '../helpers/settings/presetCustomValue';
+import { maximumNumericalOrPixelOrAutoRule } from '../utilities/rules/maximumNumericalOrPixelOrAutoRule';
+import { MARGIN_DEFAULT_PLACEHOLDER } from './defaultValues';
+import { marginStyleMap, Margin } from './types';
+
+/**
+ * Returns margin settings: margin switch, margin slider, custom margin input
+ *
+ * @param options Options for the settings
+ * @param options.id Custom suffix for the setting ids
+ * @returns {SettingBlock} Returns margin settings
+ */
```

Comment:
You need to move it on top of `export const getMarginSlider` otherwise it has no effect